### ISSUE TITLE
some missing ort files

### ISSUE THREAD

**Issue by D4RKSCR1PTS** [Wikidepia/hektCaptcha-extension#95]
here are some ort files that you don't have yet:
woodpecker
violin

**Comment by Wikidepia** [comment]
I dont have that challenges since yesterday, can you send me sitekey?

**Comment by Yishaqel** [comment]
Most of the captcha's that appear are here not working, I don't know why
Captcha key 82eeb206-6821-4595-b9c2-4ad4d32cc34d

**Comment by D4RKSCR1PTS** [comment]
> I dont have that challenges since yesterday, can you send me sitekey?

a3a5a9a9-7210-4dc7-a7bc-39c3fc73143e
host is rbxgold.com

**Comment by Yishaqel** [comment]
Is there any way to help it recognise the images by training it or something? Lately it doesn't recognise any of the images that appear.

**Comment by Wikidepia** [comment]
> Is there any way to help it recognise the images by training it or something? Lately it doesn't recognise any of the images that appear.

Not really, the new "Please click the largest animal" changes frequently might need update extension to solve it

**Comment by Yishaqel** [comment]
Yeah apparently there are quite a few recent updates to the images and it's not recognizing them
